src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -533,6 +533,44 @@ private void RemoveGroupFromGroup(object parameters)
             Analytics.TrackEvent(Actions.RemovedFrom, Categories.GroupOperations, "Group");
         }
 
+        private void DissolveNestedGroups(object parameters)
```

Comment:
Is this required? This is used by some commands not cherry-picked?

Reply:
Thanks for catching that. Cherry-pick goofiness. I went back through all the commits and compared with the cherry-picked PRs again, looks like this is the only addition.